.pre-commit-config.yaml

```diff
@@ -2,9 +2,14 @@
   sha: master
   hooks:
   - id: trailing-whitespace
+  - id: flake8
+    args: [--max-line-length=120]
```

Comment:
Array representations here differs from `name-tests-test` hook. Pick one style and follow it everywhere